### PR TITLE
feat: scan docker containers with snyk

### PR DESCRIPTION
**Pull request by revolunet**
ajoute un scan de vuln snyk sur les images docker avec allow_failure=True

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-- "Prepare"
-- "Code Quality"
-- "Registration"
-- "Data"
-- "Deploy"
-- "Notify Finished Deployment"
+  - "Prepare"
+  - "Code Quality"
+  - "Registration"
+  - "Vuln scanner"
```
Comment (lionelB) [inline .gitlab-ci.yml:18]: VulScanner should be after data since it also a registration stage

Comment (revolunet) [reply .gitlab-ci.yml:18]: data is not exposed so not sure this is really useful

Comment (lionelB) [reply .gitlab-ci.yml:18]: oki !

**Comment by revolunet** [comment]
approved ?

**Review by lionelB — APPROVED** [review]

**Pull request merged**